README.md

```diff
@@ -64,6 +64,11 @@ timesync_min_sources: 1
 # or enabled service will be configured. If no service is active or enabled, a
 # package specific to the system and its version will be selected.
 timesync_ntp_provider: chrony
+
+# List of custom settings passed through to ntp.conf.  If not defined,
+# setting is ignored.  Settings should be one per line (list item).
+timesync_ntp_custom_settings:
+  - "tinker panic 0"
```

Comment:
How idempotent is this? For example, if the user specifies
```yaml
timesync_ntp_custom_settings:
  - "tinker panic 0"
  - "a b c"
```
then sometime later specifies
```yaml
timesync_ntp_custom_settings:
  - "a    b    c"
  - "tinker panic 0"
```
Will that generate a change?  Does it matter if that generates a change?  For example, that will cause the ntp services to restart - is that ok?

Reply:
Do you want to allow the user to perform incremental changes?  For example, if the configuration already has
```yaml
timesync_ntp_custom_settings:
  - "tinker panic 0"
  - "a b c"
```
and the user wants to remove "a b c" - how to do that?  Or the user wants to remove "a b c" and add "b c a"?  For the kernel_settings role we had a similar requirement, so we documented it - https://linux-system-roles.github.io/documentation/incremental_settings.html